component/src/main/ballerina/ballerina/data/mongodb/natives.bal

```diff
@@ -1,14 +1,28 @@
 package ballerina.data.mongodb;
 
 public struct ConnectionProperties {
+    string url;
+    string username;
+    string password;
     string readConcern;
     string writeConcern;
     string readPreference;
     boolean sslEnabled;
+    boolean sslInvalidHostNameAllowed;
     int socketTimeout = -1;
     int connectionTimeout = -1;
-    int connectionsPerHost = -1;
+    int maxPoolSize = -1;
     int serverSelectionTimeout = -1;
+    int maxIdleTime = -1;
+    int maxLifeTime = -1;
+    int minPoolSize = -1;
+    int waitQueueMultiple = -1;
+    int waitQueueTimeout = -1;
+    int localThreshold = -1;
+    int heartbeatFrequency = -1;
+    string authSource;
```

Comment:
Shall we move these string parameters to the to the (i.e. after readPreference).  IMO, Grouping them according to the type will make it more cleaner and easier when accessing in underline code.

Reply:
+1. Will do the change